tests/std/tests/P0355R7_calendars_and_time_zones_formatting/test.cpp

```diff
@@ -884,6 +888,13 @@ void test_zoned_time_formatter() {
     assert(format(STR("{:%g %G %U %V %W}"), zt) == STR("21 2021 16 16 16"));
 }
 
+template <typename CharT>
+void test_localized() {
+    auto loc = locale("ru_RU");

```

Comment:
```suggestion
    auto loc = locale("ru-RU");
```
Older OSs (on Microsoft's internal test machines) accepts `"-"` but not `"_"` in locale names. See #1240.